core/dbt/contracts/files.py

```diff
@@ -228,6 +228,7 @@ class SchemaSourceFile(BaseSourceFile):
     groups: List[str] = field(default_factory=list)
     # node patches contain models, seeds, snapshots, analyses
     ndp: List[str] = field(default_factory=list)
+    semantic_models: List[str] = field(default_factory=list)
```

Comment:
semantic_nodes? (see other comment)

Reply:
@gshank To clarify, do you think we should change the name in the yml file, or just for our internal property names?

Reply:
Just for for the internal property names. I think "semantic_models" in the yaml files is fine, it matches the "models" section we already have. Naming is so hard.